argilla-server/src/argilla_server/contexts/hub.py

```diff
@@ -102,6 +102,8 @@ def _batch_index_to_row(self, batch: dict, index: int) -> dict:
                     row[feature_name] = None
                 else:
                     row[feature_name] = feature.int2str(value)
+            elif isinstance(feature, features.Sequence) and isinstance(feature.feature, features.ClassLabel):
```

Comment:
Here I understand that inside a feature that it's a `Sequence` we have inside another feature. But I'm not 100% sure.

Reply:
Ok, then my code it's fine.